lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -12,8 +12,8 @@
 #include "cling/Interpreter/Interpreter.h"
 #include "cling/Interpreter/Transaction.h"
 #include "cling/Interpreter/Value.h"
-
 #include "cling/Utils/AST.h"
+#include "cling/Utils/Output.h"
```

Comment:
You were right to note it should be removed. Just saying it does come into existence in an early commit in the PR, but was not taken out when it should have been later.